### ISSUE TITLE
Feature Request: sycn when start

### ISSUE THREAD

**Issue by zyf0330** [1995eaton/chromium-vim#686]
I think auto sync when chrome starts is good. Otherwise we must click sync manually on every machine.

**Comment by Armog** [comment]
+

**Comment by BlackCapCoder** [comment]
I disagree with this, I would much rather update manually than have slower startups. Ideally chromium would offer a `--update-extensions` flag

**Comment by zyf0330** [comment]
Are we talking about one thing?
Actually this feature shouldn't slow down boot speed of cvim.

**Comment by BlackCapCoder** [comment]
If we are talking about automatically updating the cvim extension on browser startup then this does slow down browser startup. The browser needs the new version of cvim before it can start, which is to be downloaded, which is slow.

If syncing is something different like syncing configs that can be done asynchronously, then disregard my previous comment.

**Comment by zyf0330** [comment]
Browser will update extension automatically, not controlled by extension self.
We are talking about syncing config of cvim.

**Comment by BlackCapCoder** [comment]
Gotcha. I don't mind that if it is configurable.


My preferred way to do this would be to symlink the config file to my dotfile folder, throw that on a git, and then sync everything every now and again. If you really wanted to you could throw a `git pull` before starting your browser.
